Sources/FPSStatusBarViewController.swift

```diff
@@ -15,56 +15,34 @@ class FPSStatusBarViewController: UIViewController {
 
     fileprivate let fpsCounter = FPSCounter()
     private let label = UILabel()
-
-
-    // MARK: - Initialization
-
-    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
-        super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
-
-        self.commonInit()
-    }
-
-    required init?(coder decoder: NSCoder) {
-        super.init(coder: decoder)
-
-        self.commonInit()
-    }
-
-    private func commonInit() {
-        NotificationCenter.default.addObserver(self,
-            selector: #selector(FPSStatusBarViewController.updateStatusBarFrame(_:)),
-            name: UIApplication.didChangeStatusBarOrientationNotification,
-            object: nil
-        )
+    
+    override func viewWillTransition(to size: CGSize,
+                                     with coordinator: UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
+        coordinator.animate(alongsideTransition: { [weak self] _ in
+            self?.updateStatusBarFrame()
+        }, completion: nil)
     }
 
-    deinit {
-        NotificationCenter.default.removeObserver(self)
-    }
-
-
     // MARK: - View Lifecycle and Events
 
     override func loadView() {
         self.view = UIView(frame: CGRect(x: 0.0, y: 0.0, width: 100.0, height: 100.0))
 
-        let font = UIFont.boldSystemFont(ofSize: 10.0)
+        let font = UIFont.italicSystemFont(ofSize: 12)
         let rect = self.view.bounds.insetBy(dx: 10.0, dy: 0.0)
-
-        self.label.frame = CGRect(x: rect.origin.x, y: rect.maxY - font.lineHeight - 1.0, width: rect.width, height: font.lineHeight)
+        let statusBarFrame = getStatusBarFrame()
+        let labelOriginX = statusBarFrame.height > 24 ? rect.origin.x : (statusBarFrame.width/2.0) + 30
```

Comment:
I think we could make the X position 1/4 across the width of status bar. ie centered between the left edge and center. That would then be centered under time on notch phones and work on both iPhone and iPad in a consistent way.